### DISCUSSION TITLE
Can't get zoxide working on BASH

### DISCUSSION THREAD

**Discussion by tiklup11** [ajeetdsouza/zoxide#391] [Q&A] [answered]
After installing : 

I included following in ~/.bashrc
```
export _ZO_DATA_DIR=$HOME/.local/share
export _ZO_ECHO=1
eval "$(zoxide init bash)"
```
then i ran 
`zoxide init --cmd z --hook pwd bash`

What am I doing wrong here ?

**Comment by ajeetdsouza** [comment]
After making the change in your bashrc, you can just open a new shell and use the `z` command. You don't need to call zoxide explicitly after that.

**Comment by tiklup11** [comment, marked as answer]
My bad, Manjaro uses zsh by default. I was unaware of that. Working fine now.
